index.html

```diff
@@ -26,182 +26,184 @@
         </div>
     </nav>
 </header>
-        <section class="main">
-            <div class="top-heading">
-                <h1>Hello, My name is Abbas Glad to see you!</h1>
-            </div>
-            <div class="heading-description">
-                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
-            </div>
-            <div class="touch-links">
-                <h3><a href="#">LET'S CONNECT</a></h3>
-                <div class="social-bar">
-                    <ul>
-                        <li><a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a></li>
-                        <li><a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedIn.png" alt="LinkedIn logo"></a></li>
-                        <li><a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a></li>
-                        <li><a href="#" class="connect"><img src="image-resources/M.png" alt=" logo"></a></li>
-                        <li><a href="#" class="connect"><img src="image-resources/new.png" alt=" logo"></a></li>
-                    </ul>
-                </div>
-            </div>
-        </section>
-
-        <section class="work-section">
-            <div class="work-imgs">
-                <div class="work-bg"></div>
-            </div>
-            <div class="work-topic">
-                <h2>Tonic</h2>
-            </div>
-            <div class="work-details">
+<main>
+    <section class="main">
+        <div class="top-heading">
+            <h1>Hello, My name is Abbas Glad to see you!</h1>
+        </div>
+        <div class="heading-description">
+            <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+        </div>
+        <div class="touch-links">
+            <h3><a href="#">LET'S CONNECT</a></h3>
+            <div class="social-bar">
                 <ul>
-                    <li>∘ CANOPY</li>
-                    <li class="work-2nd">∘ BACK END DEV</li>
-                    <li class="work-2nd">∘ 2023</li>
+                    <li><a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a></li>
+                    <li><a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedIn.png" alt="LinkedIn logo"></a></li>
+                    <li><a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a></li>
```

Comment:
You seem to have made an error linked the image for the LinkedIn logo. The `linkedIn.png` does not exist in the `image-resources` folder.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219592514-db69fc3a-72af-473a-aa4a-08afe85fb04f.png)